routers/api/v1/repo/issue.go

```diff
@@ -161,6 +161,12 @@ func EditIssue(ctx *context.APIContext, form api.EditIssueOption) {
 		ctx.Error(500, "UpdateIssue", err)
 		return
 	}
+	if form.State != nil {
+		if err = issue.ChangeStatus(ctx.User, ctx.Repo.Repository, strings.Compare("closed", *form.State) == 0); err != nil {
```

Comment:
I think it should compare with https://github.com/gogits/go-gogs-client/blob/master/issue.go#L16